test/framework/systemtools.py

```diff
@@ -380,6 +382,17 @@ def test_os_version(self):
         os_version = get_os_version()
         self.assertTrue(isinstance(os_version, basestring) or os_version == UNKNOWN)
 
+    def test_gcc_version_native(self):
+        """Test getting gcc version."""
+        gcc_version = get_gcc_version()
+        self.assertTrue(isinstance(gcc_version, basestring) or gcc_version == UNKNOWN)
+
+    def test_gcc_version_linux(self):
+        """Test getting gcc version (mocked for Linux)."""
+        st.get_os_type = lambda: st.LINUX
+        st.run_cmd = mocked_run_cmd
+        self.assertEqual(get_gcc_version(), '5.1.1')
+
```

Comment:
please add a test for Darwin too, to make sure `get_gcc_version` 'works' there too?